src/components/CheckboxInput/components/Checkbox.tsx

```diff
@@ -41,6 +41,10 @@ export interface CheckboxProps extends Omit<BoxProps, 'radius' | 'background' |
    * If the input should be disabled and not focusable.
    */
   isDisabled?: boolean;
+  /**
+   * If the radio input should be hidden to make way for a custom radio.
+   */
+  isHidden?: boolean;
```

Comment:
I think the name could be more specific, as it makes it look like it would hide the entire Checkbox. Perhaps 'hideIcon'?

Reply:
disregard